src/test/compile-fail/issue-13352.rs

```diff
@@ -23,4 +23,6 @@ fn main() {
         unsafe { libc::exit(0 as libc::c_int); }
     });
     2_usize + (loop {});
+    //~^ ERROR E0277
```

Comment:
Hmm.  I don't think you should have moved this test into compile-fail.

Were you intending to make a copy of it?

Could you fix the test by explicitly forcing the type of the looping expression, e.g.:
```
(2 + { let x: i8 = loop { }; x });
```